src/testrel.jl

```diff
@@ -40,10 +53,18 @@ function delete_test_database(name::String)
 end
 
 """
-    test_expected(expected::AbstractDict, results, testname)
+    test_expected(expected::AbstractDict, results, testname::String)
```

Comment:
`test_expected` is an internal method, so we might not want a docstring here, but rather just a comment

Reply:
Hmm, true. I was thinking that I wanted the method to be internal, but easy to find and understand. That doesn't require a docstring, though.